test/CappedCrowdsale.js

```diff
@@ -62,7 +70,7 @@ contract('CappedCrowdsale', function ([_, wallet]) {
   describe('ending', function () {
 
     beforeEach(async function () {
-      await advanceToBlock(this.startBlock - 1)
+      await increaseTime(moment.duration(1, 'week'))
```

Comment:
The main reason holding me back from implementing `increaseTimeTo` was not to give an illusion that we can precisely set evm timestamp to a given value. However, I think that the benefits still outweigh risks, so I've decided to introduce the helper method providing a warning about correct usage in the comments. Thanks for the feedback and discussion.